modelforge/tests/test_potentials.py

```diff
@@ -1107,3 +1107,37 @@ def test_loading_from_checkpoint_file():
 
     model = load_inference_model_from_checkpoint(chkp_file)
     assert model is not None
+
+
+@pytest.mark.parametrize(
+    "potential_name", _Implemented_NNPs.get_all_neural_network_names()
+)
+def test_saving_torchscript(potential_name, single_batch_with_batchsize, prep_temp_dir):
+    batch = single_batch_with_batchsize(
+        batch_size=1, dataset_name="QM9", local_cache_dir=str(prep_temp_dir)
+    )
+    from modelforge.potential.potential import NeuralNetworkPotentialFactory
+    from modelforge.utils.misc import load_configs_into_pydantic_models
+
+    config = load_configs_into_pydantic_models("ani2x", "qm9")
```

Comment:
It's fine to hardcode the dataset (qm9), but I think the potential_name should be passed here from the pytest parameterized input signature (otherwise the parameter doesn't impact the test and it's only testing for ani2x)

Reply:
Good catch.  This was just a typo I fixed locally but apparently never committed/pushed my change on. The intent was to check all potentials to ensure we can write and read...not just test ani2x 7 times.